prerendering.bs

```diff
@@ -545,6 +550,15 @@ Patch the [=navigate=] algorithm to prevent certain navigations in a [=prerender
   1. Otherwise, if <var ignore>browsingContext</var> is a [=prerendering browsing context=], then return.
 </div>
 
+<div algorithm=The worker's lifetime patch">
```

Comment:
This should get a new `<h3>` instead of being part of "Preventing forbidden navigations"

Reply:
Done